OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -220,20 +220,24 @@ public bool IsValidAgainst(FrozenActor victim, Actor firedBy)
 		}
 
 		/// <summary>Applies all the weapon's warheads to the target.</summary>
-		public void Impact(Target target, WarheadArgs args)
+		public void Impact(in Target target, WarheadArgs args)
 		{
 			var world = args.SourceActor.World;
 			foreach (var warhead in Warheads)
 			{
 				if (warhead.Delay > 0)
-					world.AddFrameEndTask(w => w.Add(new DelayedImpact(warhead.Delay, warhead, target, args)));
+				{
+					// Lambdas can't use in variables, so capture a copy for later
```

Comment:
"be used" instead of "use"?

Reply:
Nevermind, got the grammar now.

Reply:
Maybe change these to 
> // Lambdas can't use "in" variables, so capture a copy for later

to avoid confusion?

Reply:
That would be better, yes.